components/ShowNutritionalTable.tsx

```diff
@@ -67,7 +82,8 @@ export default function ShowNutritionalTable() {
           <thead>
             <tr>
               <th>Nutriment</th>
-              <th>pour 100 g</th>
+              <th>Valeur initiale (pour 100 g)</th>
+            <th>Valeur modifiée (pour 100 g)</th>
```

Comment:
Same here, the `<td/>` could be rendered conditionally to the existence of nutriments data